addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4404,8 +4404,8 @@ registry.sizing = SnippetOptionWidget.extend({
             if (rowEl.classList.contains('o_grid_mode')) {
                 self.options.wysiwyg.odooEditor.observerUnactive('displayBackgroundGrid');
                 backgroundGridEl = gridUtils._addBackgroundGrid(rowEl, 0);
-                self.options.wysiwyg.odooEditor.observerActive('displayBackgroundGrid');
                 gridUtils._setElementToMaxZindex(backgroundGridEl, rowEl);
+                self.options.wysiwyg.odooEditor.observerActive('displayBackgroundGrid');
```

Comment:
The fix make sense but I can't see the issue functionally. I play with `.o_handle` but I don't have too much history step... Do you have steps to reproduce the issue ?
=> If yes: add it to the commit message.
=> If no: why we don't have an history step before the change you propose ?

Reply:
@Guillaume-gdi AFAIK, there's no functional issue. It's just a line move to make the code more logical and avoid potential errors/conflicts at some point in the future.
As QSM stated:

> probably no bug because of that because changing the zindex of an element that was not observed in the first place

I can add that explanation to the commit message if you want. ;)

Reply:
I think it's ok :+1: Can you just rebase, then I will merge. Thanks :)